ADOPTERS.md

```diff
@@ -1,12 +1,35 @@
+
+## Organizations using Kubeapps
+
+<a href="https://www.coreweave.com/" border="0" target="_blank"><img alt="CoreWeave" src="./site/content/docs/latest/img/adopters/coreweave.svg" height="50"></a>
+
+<a href="https://www.ledger.com/" border="0" target="_blank"><img alt="Ledger" src="./site/content/docs/latest/img/adopters/ledger.svg" height="50"></a>
+
+<a href="https://www.terasky.com/" border="0" target="_blank"><img alt="Terasky" src="./site/content/docs/latest/img/adopters/terasky.svg" height="30"></a>
+
+## Sucess Stories
+
+Below is a list of adopters of Kubeapps that have publicly shared the details of how they use it (in alphabetical order).
+
+**[CoreWeave](https://www.coreweave.com/)**
+CoreWeave is a specialized cloud provider, built for GPU-accelerated use cases. Our infrastructure is Kubetnetes-native, and we invest in a wide range of open-source Kubernetes projects making it easy for engineers to benefit from bare metal performance without the infrastructure overhead. Kubeapps hosts our suite of 1-click deploy applications, enabling our clients to quickly and easily consume the scale of compute they need.
+
+**[Ledger](https://www.ledger.com/)**
+
+Kubeapps eases the deployment of our internal Helm charts for everybody, including those that are not familiar with Kubernetes. The ability to seamlessly integrate the Helm values with the web page (basic form support) is a killer-feature for us.
+
+**[Terasky](https://www.terasky.com/)**
+
+TeraSky is a Global Advanced Technology Solutions Provider. We utilize Kubeapps in many of our customers environments to provide a clean and simple UI for deploying and managing applications on kubernetes. we heavily use the multi cluster support in Kubeapps. We also heavily utilize the Carvel Packaging support in kubeapps, in order to provide a great solution for our customers in airgapped environments as well.
 
 ## Adding your organization to the list of adopters
 
 If you are using Kubeapps and would like to be included in the list of Kubeapps Adopters it is possible through the following options:
 
-- **Send a PR**: add an SVG version of your logo that is less than 150 KB to the [logo directory](https://github.com/vmware-tanzu/kubeapps/tree/main/docs/logos) in this repo and submit a pull request with your change including 1-2 sentences describing how your organization is using Kubeapps. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.svg).
+- **Send a PR**: add an SVG/PNG version of your logo that is less than 150 KB to the [img/adopters directory](./site/content/docs/latest/img/adopters/) in this repo and submit a pull request with your change including 1-2 sentences describing how your organization is using Kubeapps. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.svg).
```

Comment:
>logo that is less than 150 KB to the ....
Do we really need to set this limitation? I mean, we can later work on reducing the size or whatever, but 150KB (although it is a legit threshold) perhaps can bring some headaches to users with hi res logos? Not sure, just wondering

Reply:
+1